samples/asgardeo-react-app/src/error-boundary.tsx

```diff
@@ -2,6 +2,8 @@ import { AsgardeoAuthException } from "@asgardeo/auth-react";
 import React, { FunctionComponent, ReactElement } from "react";
 import { AuthenticationFailure } from "./pages/AuthenticationFailure";
 import { InvalidSystemTimePage } from "./pages/InvalidSystemTime";
+import {IssuerClaimValidationFailure} from "./pages/IssuerClaimValidationFailuer";
```

Comment:
```suggestion
import { IssuerClaimValidationFailure } from "./pages/IssuerClaimValidationFailuer";
```

Reply:
fixed in commit https://github.com/asgardeo/asgardeo-auth-react-sdk/commit/2ded725d7c96494fedec39cb54bdd58556460ce6